feat: Create a new debugging page (#4730)
Kotlin/kotlinx.coroutines: docs/topics/coroutines-and-channels.md

[//]: # (title: Coroutines and channels − tutorial)

> > We plan to revise this tutorial. For an up-to-date guide to getting started with coroutines, see [Coroutines basics](coroutines-basics.md).

Reviewer: ```suggestion
> An upcoming update will revise this tutorial. In the meantime, for an up-to-date guide to getting started with coroutines, see [Coroutines basics](coroutines-basics.md).
```
To avoid "we".

Author: sure  works for me 👍  although for these messages/notes, I generally don't mind the "we". (it's not unclear like in tutorials who the we refers to)
